xbmc/windowing/X11/WinSystemX11.cpp

```diff
@@ -20,8 +20,6 @@
 
 #include "system.h"
 
-#if defined(HAVE_X11)
```

Comment:
This breaks build on Pi. I assume this is being built unconditionally and using HAVE_X11 to avoid build issues on platforms without X.
```
/home/dc4/xbmc_holder/xbmc/xbmc/windowing/X11/WinSystemX11.h:31:22: fatal error: X11/Xlib.h: No such file or directory
 #include "X11/Xlib.h"
```